### PR TITLE
chore: contract address and config related changes for europa mainnet

### PR DESCRIPTION
**Pull request by rajkharvar**

### CODE REVIEW

File: packages/sx.js/src/evmNetworks.ts [hunk 1]
```diff
@@ -217,6 +217,35 @@ export const evmNetworks = {
       Axiom: null,
       Isokratia: null
     }
+  },
+  'europa-mainnet': {
```
Comment (yohanelly95) [inline packages/sx.js/src/evmNetworks.ts:221]: @rajkharvar this was updated for testing right? We need to redeploy this with razor deployer address next correct?

Comment (rajkharvar) [reply packages/sx.js/src/evmNetworks.ts:221]: yes

File: packages/sx.js/src/utils/merkleVoting.ts [hunk 2]
```diff
@@ -3,8 +3,8 @@ import { Contract } from '@ethersproject/contracts';
 
 import MerkleVotingABI from '../abis/MerkleVoting.json';
 
-const MERKLE_VOTING_RPC_PROVIDER = 'https://mainnet.skalenodes.com/v1/turbulent-unique-scheat';
-const MERKLE_VOTING_API = 'https://sx-api.razorscan.io';
+const MERKLE_VOTING_RPC_PROVIDER = 'https://mainnet.skalenodes.com/v1/elated-tan-skat';
+const MERKLE_VOTING_API = 'http://localhost:3000';
```
Comment (yohanelly95) [inline packages/sx.js/src/utils/merkleVoting.ts]: this change needs to be reverted @rajkharvar

Comment (rajkharvar) [reply packages/sx.js/src/utils/merkleVoting.ts]: yes, since tested by running the setup locally

Comment (rajkharvar) [reply packages/sx.js/src/utils/merkleVoting.ts]: reverted in [77b16be](https://github.com/razor-network/sx-monorepo/pull/13/commits/77b16bee729ce7bc15ecee260ba42847c6f997c5)

File: packages/sx.js/src/evmNetworks.ts (hunk 1 above)
Comment (rajkharvar) [reply packages/sx.js/src/evmNetworks.ts:221]: reverted in [d7291cb](https://github.com/razor-network/sx-monorepo/pull/13/commits/d7291cbfb76ecbf28fc786dce4643b22519ad659)

File: apps/ui/src/networks/evm/constants.ts [hunk 3]
```diff
@@ -18,14 +18,14 @@ import { MAX_SYMBOL_LENGTH } from '@/helpers/constants';
...
+export const MERKLE_VOTING_STRATEGY_ADDRESS = evmNetworks['europa-mainnet'].Strategies.MerkleVoting;
 export const COOLDOWN_PERIOD = 10; // * seconds
 export const MAX_ACTIVE_PROPOSALS = 100;
 // !NOTE: Update this after space deployed
-export const EXECUTION_STRATEGY_ADDRESS = '0xdF2Fb38249640C1D7ee6f0347970C4D3059f8D63';
-export const QUORUM = '10000000000000000000000000'; // * 10 million
-export const SPACE_ADDRESS = '0x579f0121f678Da51a66fFC0bf6059fF87f9045E5';
-export const REDIRECT_URL = `/skale-mainnet:${SPACE_ADDRESS}`;
+export const EXECUTION_STRATEGY_ADDRESS = '0x7659C8469c50218EfC881382A85D919aCa22b200';
+export const QUORUM = '100000000000000000000000'; // * 10 million
```
Comment (yohanelly95) [inline apps/ui/src/networks/evm/constants.ts]: @rajkharvar this is not 10M

File: apps/ui/src/networks/evm/index.ts [hunk 4]
```diff
@@ -88,6 +88,13 @@ export const METADATA: Record<string, Metadata> = {
     apiUrl: 'https://sx-indexer.razorscan.io/subgraphs/name/snapshot-labs/sx-subgraph',
     avatar: 'https://ipfs.io/ipfs/QmRXoxMnz9ViWzQ7Ewg7dA8XtQoxp8gQ4cWjqj1Vk2ue2p/razor.png',
     blockTime: 10.7 // * Calculated from the average block time of the last 1000, 10000 blocks
+  },
+  'europa-mainnet': {
+    name: 'Europa Mainnet',
+    chainId: 2046399126,
+    apiUrl: 'http://localhost:8000/subgraphs/name/snapshot-labs/sx-subgraph',
```
Comment (yohanelly95) [inline apps/ui/src/networks/evm/index.ts]: can update this as well

File: apps/ui/src/networks/evm/constants.ts (hunk 3 above)
Comment (rajkharvar) [reply apps/ui/src/networks/evm/constants.ts]: updated in [519e865](https://github.com/razor-network/sx-monorepo/pull/13/commits/519e865f9396ea8204585c5fdef1794b166d4d8a)

File: apps/ui/src/networks/evm/index.ts (hunk 4 above)
Comment (rajkharvar) [reply apps/ui/src/networks/evm/index.ts]: updated in [519e865](https://github.com/razor-network/sx-monorepo/pull/13/commits/519e865f9396ea8204585c5fdef1794b166d4d8a)

**Review by yohanelly95 — APPROVED** [review]

**Pull request merged**